resources/user-agents/browsers/firefox-on-android/firefox-android-46-to-47.json

```diff
@@ -33,7 +33,7 @@
             "AndroidMobile_2_3",
```

Comment:
please flip the sorting the platforms here, like did in the `firefox-android-48-on.json` file

Reply:
flipped